plugin/src/test/java/org/opensearch/ml/tools/CatIndexToolIT.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.tools;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+import org.junit.Before;
+import org.opensearch.client.Response;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.ml.rest.RestBaseAgentToolsIT;
+import org.opensearch.ml.utils.TestHelper;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+public class CatIndexToolIT extends RestBaseAgentToolsIT {
+    private String agentId;
+    private final String question = "{\"parameters\":{\"question\":\"please help list all the index status in the current cluster?\"}}";
+
+    @Before
+    public void setUpCluster() throws Exception {
+        registerCatIndexFlowAgent();
+    }
+
+    private List<String> createIndices(int count) throws IOException {
+        List<String> indices = new ArrayList<>();
+        for (int i = 0; i < count; i++) {
+            String indexName = "test" + i;
```

Comment:
As mentioned, this "test" string plus an incrementing integer can be the prefix of other strings and be less useful for testing matching using substrings (`contains()` or `startswith()`).  If you're not using delimiters in the test it's important to make all of these unique substrings.  Is it possible to use a random string here?

Reply:
Same above, I'm open to change to random string if you have a concrete example.

Reply:
I was thinking an example could show that the current test can't cover, anyway it's not a big change so I can make this change.

Reply:
I already gave the example where you were testing for “test1” and your current test would have passed for “test10” or “teat100”. I’m not in favor of having tests that pass when they should fail. 

It’s just a suggestion.  Using an end of word delimiter would be equally effective.

Reply:
changed to random string